src/shared/locales/en/translation.json

```diff
@@ -57,6 +59,7 @@
         "share": "Share",
         "proceed": "Proceed",
         "qr": "QR",
+        "of": "of",
```

Comment:
If possible we should use [interpolation](https://www.i18next.com/translation-function/interpolation) or the [`Trans` component](https://react.i18next.com/latest/trans-component) here so that translators have better context. I can make a PR for this

Reply:
PR: #2452